Formula/dish.rb

```diff
@@ -0,0 +1,23 @@
+class Dish < Formula
+  desc "Tool for parallel sysadmin of multiple hosts"
+  homepage "https://freeshell.de/~drimiks/gnu/dish.shtml"
```

Comment:
Is this actively maintained? I couldn't see when that version was released, or where development takes place (bug reports, code tracker, etc)

Reply:
The last release was in 2013.